adapters/ing-psd2-adapter/src/main/java/de/adorsys/xs2a/adapter/service/ing/IngPsd2AccountInformationService.java

```diff
@@ -144,21 +144,16 @@ private String getAccessToken(Map<String, String> headers) {
     }
 
     @Override
-    public Response<TransactionsResponse> getTransactions(String accountId,
-                                                          Map<String, String> queryParameters,
-                                                          Map<String, String> headers) {
-        return getTransactions(accountId, new QueryParameters(queryParameters), new Headers(headers));
-    }
-
-    private Response<TransactionsResponse> getTransactions(String accountId,
-                                                           QueryParameters queryParameters,
-                                                           Headers headers) {
-        String accessToken = headers.getAccessToken();
+    public Response getTransactions(String accountId,
+                                    Map<String, String> queryParameters,
+                                    Map<String, String> headers) {
+        QueryParameters queryParameters1 = new QueryParameters(queryParameters);
```

Comment:
I propose to come up with a better naming to a variable `queryParameters1` by either renaming `Map<String, String> queryParameters` to `queryParametersMap` or renaming `queryParameters1` to smth like ``queryParametersHolder` (or any other meaningful naming).

Reply:
I didn't notice the variable rename after method inline refactoring. Reverted.